pkg/etcd/config.go

```diff
@@ -22,3 +22,9 @@ type EtcdConfig struct {
 	// List of etcd server endpoints
 	Endpoints []string `json:"endpoints" validate:"required,gt=0,dive,hostname_port|url|fqdn"`
 }
+
+// EtcdConfigOverride is provided by FluxNinja plugin if EnableCloudController flag is set.
+type EtcdConfigOverride struct {
+	EtcdConfig
+	APIKey string `json:"api_key"`
+}
```

Comment:
Nit: would be nice to avoid mentioning api key in aperture (make the struct more generic like "header key, header val" :shrug:)